exopy_hqc_legacy/tasks/tasks/instr/run_awg.py

```diff
@@ -10,33 +10,51 @@
 
 """
 import logging
+import time
 
-from atom.api import (Unicode, set_default)
+from atom.api import (Float, Unicode, set_default)
```

Comment:
Please replace `Unicode` by `Str`

Reply:
What's the difference?

Reply:
Since Python 2 is dead, str is Unicode and Unicode will be deprecated in the next release of Atom (to be released in a week or so).